app/konnect/org-management/system-accounts.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Manage System Accounts
+content_type: how-to
+no_version: true
+---
+
+This guide explains what a system account is, how they vary from user accounts, and how to manage a system account using the [{{site.konnect_short_name}} API](https://developer.konghq.com/spec/5175b87f-bfae-40f6-898d-82d224387f9b/fc735302-d8ac-4e66-a9ef-225569a75d3c). 
+
+System accounts are a service account in {{site.konnect_short_name}}. Because they are not associated with an email address and a user, system accounts can be used for automation and integrations. 
+
+System accounts offer the following benefits over regular user accounts:
+
+* System accounts are not associated with an email address. This allows you to use the account as part of an automation or integration that isn't associated with any person’s identity.
+* When you use a user account as part of an automation or integration and that user leaves the company, this breaks the automations and integrations. If you use a system account instead, the automations and integrations wouldn't break.
+* Because system accounts can't be assigned the same permissions as a user account, they can't access the {{site.konnect_short_name}} UI.
```

Comment:
```suggestion
* System accounts do not have sign in credentials and thus cannot access the {{site.konnect_short_name}} UI. These accounts are intended for use with APIs and CLIs.
```

Suggestion: SA's can be assigned the same roles as user accounts so I think the reasoning is incorrect. We should point out that they are for use with API and CLI.